code/modules/mob/living/carbon/human/say.dm

```diff
@@ -12,7 +12,7 @@
 		var/obj/item/clothing/mask/chameleon/V = wear_mask
 		if(V.vchange && wear_id)
 			var/obj/item/card/id/idcard = wear_id.GetID()
-			if(istype(idcard))
+			if(istype(idcard) && !istype(idcard, /obj/item/card/id/paper))
```

Comment:
I think the ID paper should work for this, considering its the mask doing the voice changing and has nothing to do with the ID

Reply:
someone else told me to explicitly remove it for balance reasons

Reply:
No, I was asking if you know what you're doing

Reply:
i think i know what im doing?? should i keep the id not working or

Reply:
Nah, literally was asking. You're taking my words more than you need to understand enough.

Reply:
im confused- should i keep the id card not working with the voicechanger??

Reply:
It is up to you at this point, and I was only letting you know the paper id is compatible with the syndi voice changer. What Bacon said is suggesting, not more than you must reflect his opinion. If you think it shouldn't work with the voice changer through the paper ID, then say your opinion here. That's it.